tests/functional/adapter/test_ephemeral.py

```diff
@@ -14,25 +11,3 @@ def test_ephemeral_multi(self, project):
         check_relations_equal(project.adapter, ["seed", "dependent"])
         check_relations_equal(project.adapter, ["seed", "double_dependent"])
         check_relations_equal(project.adapter, ["seed", "super_dependent"])
-
-        assert os.path.exists("./target/run/test/models/double_dependent.sql")
-        with open("./target/run/test/models/double_dependent.sql", "r") as fp:
-            sql_file = fp.read()
-
-        sql_file = re.sub(r"\d+", "", sql_file)
-        expected_sql = (
-            "create or replace view test_test_ephemeral.double_dependent"
-            "as"
-            "with __dbt__cte__base as ("
-            "select * from test_test_ephemeral.seed"
```

Comment:
Do you mean the table name here is different between UC and non-UC tests?

Reply:
2-layer vs. 3-layer, to be exact.

Reply:
👍  3L namespace for UC tables.